scripts/lib/CIME/SystemTests/err.py

```diff
@@ -29,3 +30,12 @@ def _case_two_custom_prerun_action(self):
         expect(len(restart_list) >= 1, "No restart files found in {}".format(rest_root))
         restore_from_archive(self._case, rest_dir=
             os.path.join(rest_root, restart_list[0]))
+
+    def _case_two_custom_postrun_action(self):
+        # Link back to original case1 name
```

Comment:
Can you add a brief comment saying why this is needed - e.g.

```python
        # This is needed so that the necessary files are present for baseline comparison and generation,
        # since some of them may have been moved to the archive directory
```